pmpro-auto-renewal-checkbox.php

```diff
@@ -39,7 +39,7 @@ function pmproarc_pmpro_membership_level_after_other_settings() {
 	$options = pmproarc_getOptions($level_id);
 ?>
 <div id="arc_setting_div">
-	<h3 class="topborder"><?php esc_html_e('Auto-Renewal Settings', 'pmpro-auto-renewal-checkbox');?></h3>
+	<h2 class="topborder"><?php esc_html_e('Auto-Renewal Settings', 'pmpro-auto-renewal-checkbox');?></h2>
```

Comment:
Reverting this change as we only want to update the frontend h3's.
```suggestion
	<h3 class="topborder"><?php esc_html_e('Auto-Renewal Settings', 'pmpro-auto-renewal-checkbox');?></h3>
```